primitives/primproc/columncommand.cpp

```diff
@@ -202,6 +202,20 @@ void ColumnCommand::_loadData()
   bpp->cachedIO += wasCached;
   bpp->physIO += blocksRead;
   bpp->touchedBlocks += blocksToLoad;
+
+  if (_hasAuxCol)
```

Comment:
JFYI The naming conventions adds underscore as a suffix not as a prefix.

Reply:
I used the existing naming convention for the other member variables in this file, such as `_isScan`.

Reply:
We have three different existing styles and decided to go with the one whatsoever. I understand that it might contradict with the style in this particular file but if we follow the style that exists(the three of them) we never reach a single style, right?

Reply:
Done. I have changed this member var with underscore as a suffix instead of a prefix.